crates/sp1/ev-hyperlane/script/Cargo.toml

```diff
@@ -12,17 +12,22 @@ path = "src/bin/main.rs"
 name = "vkey-ev-hyperlane"
 path = "src/bin/vkey.rs"
 
+[features]
+retry = []
```

Comment:
Idk if we need the feature flag for this on this binary, do we? It seems like it just adds a bit more operational overhead for having to compile things correctly etc. 

I think we could just include this without the feature flag. What do you think?